static/js/client.js

```diff
@@ -963,9 +969,9 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
         })
         .attr('fill', function (d) { return d.color; })
         .style('opacity', function (d) { return highlightBrushPoints(d) })
-        .attr('stroke-width', function (d) {if (d.type == 'mbg') return 2; else return 0; })
-        .attr('stroke', function (d) { return 'white'; })
-        .attr('r', function(d) { if (d.type == 'mbg') return 4; else return 2;});
+        .attr('stroke-width', function (d) { return d.type == 'mbg' ? 2 : 0; })
+        .attr('stroke', function ( ) { return 'white'; })
+        .attr('r', function (d) { return d.type == 'mbg' ? 4 : 2; });
 
       if (badData.length > 0) {
         console.warn("Bad Data: isNaN(sgv)", badData);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/761330330/issues/source?bid=2452379&fileBranchId=2452987#l977)